packages/babel-helper-modules/src/index.js

```diff
@@ -231,6 +246,48 @@ const reexportGetter = template(`
 `);
 
 /**
+ * Build a statement declaring a variable that contains all of the exported
+ * variable names in an object so they can easily be referenced from an
+ * export * from statement to check for conflicts.
+ */
+function buildExportNameListDeclaration(
+  programPath: NodePath,
+  metadata: ModuleMetadata,
+) {
+  const exportedVars = Object.create(null);
```

Comment:
This is used like a Set so it should be a Set 🤔

Reply:
I pass it to `t.valueToNode` which is why I kept it as an object.